src/components/organisms/NewProfileModal/components/ProfileModalBadges/ProfileModalBadges.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import { User } from "types/User";
+import { ContainerClassName } from "types/utility";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { Badges } from "components/organisms/Badges";
+
+interface Props extends ContainerClassName {
+  viewingUser: WithId<User>;
+  venue: WithId<AnyVenue>;
+}
+
+export const ProfileModalBadges: React.FC<Props> = ({
+  venue,
+  viewingUser,
+  containerClassName,
+}) => {
+  return (
+    <>
+      {venue?.showBadges && viewingUser && (
+        <Badges
+          containerClassName={containerClassName}
+          user={viewingUser}
+          currentVenue={venue}
+        />
+      )}
+    </>
```

Comment:
```suggestion
      {venue?.showBadges && viewingUser && (
        <Badges
          containerClassName={containerClassName}
          user={viewingUser}
          currentVenue={venue}
        />
      )}
```

Reply:
Cannot be done as the return value from the component must be JSX or smth like that.
Here is the error:
> Type 'undefined' is not assignable to type 'ReactElement<any, any> | null'.